Modules/Devices/test/Devices.Domain.Tests/Identities/DeletionProcessApprovalReminderTests.cs

```diff
@@ -103,6 +103,20 @@ public void DeletionProcessApprovalReminder3Sent_fails_when_no_deletion_process_
         acting.Should().Throw<DomainException>().Which.Code.Should().Be("error.platform.validation.device.noDeletionProcessWithRequiredStatusExists");
     }
 
+    [Fact]
+    public void GetEndOfApprovalPeriod_returns_expected_date()
+    {
+        // Arrange
+        SetupSystemTime();
+        var deletionProcess = IdentityDeletionProcess.StartAsOwner(IdentityAddress.Create(Array.Empty<byte>(), "id1"), DeviceId.Parse("DVC1"));
+
```

Comment:
I think we should manually set the approval time within this test. This increases
1. stability, because the test doesn't start failing once we decide to change the approval time.
2. readability, because the reader can reconstruct how the `endOfApprovalPeriod` is calculated.